source/dotnet/wholesale-api/Orchestrations.IntegrationTests/Functions/Calculation/CalculationOrchestrationTests.cs

```diff
@@ -269,4 +268,66 @@ public async Task MockJobsRunsGetLifeCycleScenario_WhenCallingStartCalculationEn
             ((int)CalculationState.Completed).ToString(),
         ]);
     }
+
+    /// <summary>
+    /// Verify the job status monitor (loop) breaks if we reach expiry time.
+    /// </summary>
+    [Fact]
+    public async Task MockJobsRunsGetAsRunning_WhenCallingStartCalculationEndPoint_OrchestrationCompletesWithExpectedGetJobStatusActivity()
```

Comment:
```suggestion
    public async Task MockJobsRunsGetAsRunning_WhenCallingStartCalculationEndpointAndCalculationTimeoutIsExceeded_OrchestrationCompletesWithExpectedGetJobStatusActivity()
```

I might just not understand the test naming convention, but when only reading the test name (without reading the summary comment) I don't now that this actually tests the calculation timeout. Maybe something like the above suggestion?

Reply:
I'm not sure about the test names. It seems to differ per team and even per c# component. ;)

I used to get inspiration from this: https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html
But for integration/subsystem/system tests it not that easy.